frontend/components/commons/results/ResultsList.vue

```diff
@@ -118,6 +124,13 @@ export default {
       this.scrollComponent.addEventListener("scroll", this.onScroll);
     }
   },
+  updated() {
+    const scroll = document.getElementById("scroll");
+    if (scroll) {
+      this.scrollComponent = scroll;
+      this.scrollComponent.addEventListener("scroll", this.onScroll);
+    }
+  },
```

Comment:
Is this related to rules summary  or just an old code fix? IF second case, please move this change to another PR

Reply:
this was to manage the header in the summary, but I'm going to change it